### DISCUSSION TITLE
Why i see xxxx.suspabase.co here ? i should see the application name,

### DISCUSSION THREAD

**Discussion by 736f666961** [supabase/supabase#14029] [Questions] [answered]
![vy](https://user-images.githubusercontent.com/61698433/235297857-3e31d9f9-ca56-4fca-942a-0a4d32b6f867.png)
i should see the application name, does anyone know what's the solution for this problem ?

**Comment by GaryAustin1** [comment, marked as answer]
This might help: https://github.com/orgs/supabase/discussions/2925
